### PR TITLE
Kb nav codelab

### PR DESCRIPTION
**Pull request by alschmiedt**
Create a keyboard nav codelab

### CODE REVIEW

File: codelabs/keyboard_navigation/keyboard_navigation.md [hunk 1]
```diff
@@ -0,0 +1,450 @@
...
+Keyboard navigation is the first step in making Blockly more accessible. This guide focuses on how to modify keyboard navigation for testing purposes. 
+
+### Prerequisites
+1. A basic understanding of how to use the Blockly playground locally. This can be found in `tests/playground.html`.
+1. Read through the keyboard navigation [documentation](https://developers.google.com/blockly/guides/configure/web/keyboard-nav).
+
+### What you'll learn
+In this codelab you will learn:
+1. How to change the behavior of a cursor.
+1. How to change the look of cursors and markers.
```
Comment (samelhusseini) [inline codelabs/keyboard_navigation/keyboard_navigation.md:20]: All items are 1s.

File: codelabs/keyboard_navigation/keyboard_navigation.md [hunk 2]
```diff
@@ -0,0 +1,450 @@
...
+
+## Terminology
+A **Marker** holds a location and is not movable. 
+
+A **Cursor** is a marker that can move. It extends a `Blockly.Marker` but adds logic to allow the marker to move through the blocks, inputs, fields, connections and workspace coordinates.
+
+![Displays the different parts of a block. The previous connection on the top of a block. The next connection on the bottom of a block. Input value as a cut out of a puzzle piece. The statement input as a connection inside of a block. The output connection as a puzzle piece.](./block_terms.png)
+
+## Setup
+In this codelab you will add code to the Blockly playground to create and use a new cursor. You can find the playground at `tests/playground.html`.
```
Comment (samelhusseini) [inline codelabs/keyboard_navigation/keyboard_navigation.md]: Should we add a direct link to the playground?

File: codelabs/keyboard_navigation/keyboard_navigation.md (hunk 1 above)
Comment (alschmiedt) [reply codelabs/keyboard_navigation/keyboard_navigation.md:20]: They render as a numbered list.

**Review by moniika — APPROVED** [review]

File: codelabs/keyboard_navigation/keyboard_navigation.md (hunk 2 above)
Comment (alschmiedt) [reply codelabs/keyboard_navigation/keyboard_navigation.md]: Yup, good idea

**Pull request merged**